lib/dom-jquery.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * @fileoverview DOM abstraction to use jquery to add and remove event listeners
+ * in T3
+ * @author jdivock
+ */
+
+Box.JQueryDOM = (function(){
+    'use strict';
+
+    return {
+
+		type: 'jquery',
+
+		/**
+		 * Returns the first element that is a descendant of the element
+		 * on which it is invoked that matches the specified group of selectors.
+		 * @param {HTMLElement} root parent element to query off of
+		 * @param {string} selector query string to match on
+		 *
+		 * @returns {HTMLElement} first element found matching query
+		 */
+		query: function(root, selector){
+			return $(root).find(selector)[0];
+		},
+
+		/**
+		 * Returns a non-live NodeList of all elements descended from the
+		 * element on which it is invoked that match the specified group of CSS selectors.
+		 * @param {HTMLElement} root parent element to query off of
+		 * @param {string} selector query string to match on
+		 *
+		 * @returns {HTMLElement|Array} elements found matching query
+		 */
+		queryAll: function(root, selector){
+			return $(root).find(selector);
```

Comment:
I believe this returns a jQuery object rather than an array. We should make sure it always returns an array.